scripts/output_env.sh

```diff
@@ -13,42 +13,40 @@
 # This includes:
 #   - architecture of the system
 #   - type and version of the operating system
-#   - version of armcc, gcc-arm and gcc compilers
-#   - version of libc, clang, asan and valgrind
+#   - version of armcc, clang, gcc-arm and gcc compilers
+#   - version of libc, clang, asan and valgrind if installed
 #   - version of gnuTLS and OpenSSL
 
 echo
 echo "1) Operating system and architecture:"
 uname -a
 
 echo
-if `hash armcc` > /dev/null; then
+if `hash armcc 2>/dev/null`; then
```

Comment:
Isnt it better if you do `2>&1 >/dev/null`. It might not be required in all cases, but just to be on the safe side.